notebooks/utils/notebook_utils.py

```diff
@@ -206,6 +208,8 @@ class VideoPlayer:
 
     def __init__(self, source, size=None, flip=False, fps=None, skip_first_frames=0):
         import cv2
+
+        self.cv2 = cv2  # This is done to access the package in class methods
```

Comment:
as we know which specific function we will use from module, can we jus import them and store as class attributes?

Reply:
why would we do this? The whole package will be loaded anyway + we need more fields in the class

Reply:
possibly prevent illegal usage of package inside class, when user uses field instead of direct module usage. I do not insist, just try to reduce number of questions to the code design.

BTW, what happens if uses does not install opencv at all, should we provide some warning for this case or import error will be enough clear from UX perspective?

Reply:
These are good questions. In my understanding, currently, we manage package installation directly within the example notebooks. It is the easiest approach, but I believe that the introduction of an import-checking mechanism of some kind would reduce the clarity and transparency of the code, and would be somewhat redundant than helpful.